DEPENDENCIES.md

```diff
@@ -1,30 +1,30 @@
 ### Composer
```

Comment:
Isn't this a bit inconsistent?
IMHO the package manager should ne the highest order and his packages are one level smaller.

Now we have an **h3** for manager, **h2** for normal packages and **h1** for locked one?

Reply:
Thanks for pointing it out, is fixed now! And even the tests shouldn't fail anymore..